sktime/forecasting/base/_base.py

```diff
@@ -448,6 +449,111 @@ def predict_interval(
 
         return pred_int
 
+    def predict_var(
+        self,
+        fh=None,
+        X=None,
+    ):
+        """Compute/return variance forecasts.
```

Comment:
can we explain here a bit better what a variance forecast is about? For me its not so clear yet.

Reply:
If you are making predictions of random variables `Y_1, Y_2, ... Y_N`, it is a statement about what the estimator thinks `Var(Y_1), Var(Y_2),  ..., Var(Y_N)` are.

Reply:
ah I see, maybe just add this to the docstring?

Reply:
ok, added two lines. Anything more, and we need some groundwork in mathematical notation - I´ll work on the scientific reference.